.moban.dt/requirements.txt.jj2

```diff
@@ -0,0 +1,3 @@
+{% for package, data in coala_requirements.items() | sort %}
```

Comment:
Yes. This would be simpler. So, instead of keeping a seperate `requirements.yaml` we can put these dependencies in .moban.yaml itself. Like in [here](https://github.com/coala/coala/blob/master/.moban.yaml).